sunpy/instr/aia/response.py

```diff
@@ -0,0 +1,308 @@
+"""
+Calculate SDO/AIA wavelength and temperature response functions.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import os
+import warnings
+from collections import namedtuple
+
+import numpy as np
+import h5py
```

Comment:
Hmmm, h5py is not always that easily installed ;) I am ok with requiring it for this, but I want the dependency to be limited to this import only (not aiaprep).